ODIN_II/SRC/read_xml_config_file.cpp

```diff
@@ -217,6 +219,27 @@ void read_optimizations(pugi::xml_node a_node, config_t* config, const pugiutil:
             config->split_hard_multiplier = 1;
     }
 
+    child = get_single_child(a_node, "mix_soft_hard_blocks", loc_data, OPTIONAL);
```

Comment:
can you embed that to the class HardSoftLogicMixer Multiplier type, 
that way other can duplicate that and continue on moving onto classes